src/Microsoft.PowerShell.Commands.Diagnostics/GetCounterCommand.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.PowerShell.Commands
     ///
     /// Class that implements the Get-Counter cmdlet.
     ///
-    [Cmdlet(VerbsCommon.Get, "Counter", DefaultParameterSetName = "GetCounterSet", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=138335")]
+    [Cmdlet(VerbsCommon.Get, "Counter", DefaultParameterSetName = "GetCounterSet", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2109647")]
```

Comment:
Should we update a test for the new FWLink code?

Reply:
Will add